src/impression.js

```diff
@@ -29,9 +31,13 @@ export function maybeTrackImpression(win) {
   if (!isExperimentOn(win, 'alp')) {
     return;
   }
+
   const viewer = viewerForDoc(win.document);
   /** @const {string|undefined} */
-  const clickUrl = viewer.getParam('click');
+  let clickUrl = viewer.getParam('click');
+
+  // One test clickUrl
```

Comment:
Please delete. Or add a comment here with the properly encoded click URL. Can be produced by using any example AMP doc and then appending `#click={encodeURIComponent('YOUR_URL')}`